exs/mundo_1/ruby/013.rb

```diff
@@ -0,0 +1,12 @@
+=begin
+Desafio 013
+Problema: Faça um algoritmo que leia o salário de um funcionário
+          e mostre seu novo salário, com 15% de aumento.
+Resolução do problema:
+=end
```

Comment:
Espaçamento novamente

```suggestion
=begin
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário
          e mostre seu novo salário, com 15% de aumento.
          
Resolução do problema:
=end
```